### ISSUE TITLE
Not able to edit and increment/decrement value in TextFields in case of Screen reader(Narrator) is on.

### ISSUE THREAD

**Issue by Harkesh030679** [microsoft/fluentui#12415]
Not able to edit and increment/decrement value in TextFields in case of Screen reader(Narrator) is on.

### Environment Information

OS: Win 10 Enterprise N

Browser:  Edge + Narrator

### Describe the issue:

Not able to edit and increment/decrement value in TextFields in case of Screen reader(Narrator) is on.

### Please provide a reproduction of the issue in a codepen:
You can repro it in below codepen link, Screen reader(narrator) must be on before repro
https://codepen.io/Harkesh/pen/bGdmzpR?editable=true

#### Actual behavior:

Not able to edit and increment/decrement value in TextFields in case of Screen reader(Narrator) is on.

#### Expected behavior:

Should be able to edit and increment/decrement value in TextFields in case of Screen reader(Narrator) is on.

**Comment by aneeshack4** [comment]
@Harkesh030679 Are you repro-ing this when scan mode is off?

**Comment by Harkesh030679** [comment]
When scan mode is off then it is working fine.

**Comment by aneeshack4** [comment]
@Harkesh030679 Then this is a Narrator bug due to its lack of scan mode support. Unfortunately we can't do anything on our end.
